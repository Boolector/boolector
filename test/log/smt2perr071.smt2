(declare-fun a () (Array (_ BitVec 32) (_ BitVec 8)))
(assert (=> a true)
