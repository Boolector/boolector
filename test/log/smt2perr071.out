log/smt2perr071.smt2:2:13: argument 1 of '=>' is an array term
